### PR TITLE
Make refresh_interval updatable

### PR DESCRIPTION
**Pull request by jamesgorrie**
Allows us to update the [`refresh_interval`](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#index-refresh-interval-setting) of an index.

The plan would be to set the `refresh_interval` to `-1` while reindexing, which is fine as we're generally not reindexing into and index that is a `search` index.

### CODE REVIEW

File: .bsp/sbt.json [hunk 1]
```diff
@@ -1 +1 @@
-{"name":"sbt","version":"1.4.1","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Users/jamesgorrie/Library/Java/JavaVirtualMachines/adopt-openj9-11.0.11/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/jamesgorrie/Library/Application Support/JetBrains/IntelliJIdea2021.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
\ No newline at end of file
+{"name":"sbt","version":"1.4.1","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Users/jamesgorrie/.sdkman/candidates/java/11.0.10.j9-adpt/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/jamesgorrie/Library/Application Support/JetBrains/IntelliJIdea2021.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```
Comment (jamesgorrie) [inline .bsp/sbt.json:1]: Pfft. Not sure what this is, will revert.

Comment (alexwlchan) [reply .bsp/sbt.json:1]: I think we probably want to just exclude this file in `.gitignore`.

File: elasticsearch/docker-compose.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 elasticsearch:
-  image: "docker.elastic.co/elasticsearch/elasticsearch:7.12.2"
+  image: "docker.elastic.co/elasticsearch/elasticsearch:7.12.1"
```
Comment (jamesgorrie) [inline elasticsearch/docker-compose.yml:2]: This was set to the scala4s version, there is no 7.12.2 in elasticsearch.

File: .bsp/sbt.json (hunk 1 above)
Comment (jamesgorrie) [reply .bsp/sbt.json]: Yip - removed.

**Review by kenoir — APPROVED** [review]

**Pull request merged**